client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -178,14 +178,10 @@ <h3 class="oui-tile__title" data-i18n-static="hosting_dashboard_configuration"><
                                             <div>
                                                 <strong class="d-block"
                                                         data-i18n-static="hosting_dashboard_service_ssl"></strong>
-                                                <span data-ng-bind="hosting.hasHostedSsl || ssl ? i18n.common_yes : i18n.common_no"></span>
-                                                <span class="ml-2"
-                                                      data-ng-if="ssl && ssl.status === 'creating' || ssl.status === 'deleting' || ssl.status === 'regenerating'"
-                                                      data-uib-tooltip="{{tr('hosting_dashboard_service_ssl_' + ssl.status)}}">
-                                                    <oui-spinner data-size="s"></oui-spinner>
-                                                </span>
+                                                <span data-ng-bind="ctrl.chooseCertificateStatusText()"></span>
                                             </div>
-                                            <div class="ml-auto mb-auto" style="position: relative"
+                                            <div class="ml-auto mb-auto"
+                                                 style="position: relative"
```

Comment:
Can we avoid using inline style?

Reply:
It's an old code but you're perfectly right